scripts/playbooks/roles/add_credentials/templates/aws_credential.txt.j2

```diff
@@ -0,0 +1,10 @@
+json={
+  "credentials": {
+    "scope": "GLOBAL",
+    "id": "{{ credential.value.id }}",
+    "accessKey": "{{ credential.value.accessKey }}",
+    "secretKey": "{{ credential.value.secretKey }}",
+    "description": "{{ credential.value.description }}",
+    "$class": "com.cloudbees.jenkins.plugins.awscredentials.AWSCredentialsImpl"
+  }
+}
```

Comment:
Were you able to test this against the jenkins instance by running the playbook?

Reply:
it was successfully tested by running the playbook